Javascript/q42.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q42</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    .flex {
+      display: flex;
+      flex-wrap: wrap;
+      width: 300px;
+      gap: 10px;
+    }
+
+    .box {
+      width: 200px;
+    }
+
+    button {
+      margin-top: 5px;
+    }
+  </style>
+
+  <script>
+    function myFunction(){
```

Comment:
This is not a very descriptive function name. Please improve this

Reply:
done the changes

Reply:
ok